### PR TITLE
Feature/add kafka consumer group offsets metric

### PR DESCRIPTION
**Pull request by tsardima**

### CODE REVIEW

File: src/main/java/com/icthh/xm/tmf/ms/activation/config/KafkaOffsetsMetric.java [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+import org.springframework.cloud.stream.binder.kafka.properties.KafkaBinderConfigurationProperties;
+import org.springframework.kafka.core.ConsumerFactory;
+import org.springframework.kafka.core.DefaultKafkaConsumerFactory;
+import org.springframework.stereotype.Component;
+import org.springframework.util.ObjectUtils;
+
+@Slf4j
+@RequiredArgsConstructor
+@Component
+public class KafkaOffsetsMetric implements MetricSet {
```
Comment (amedvedchuk) [inline src/main/java/com/icthh/xm/tmf/ms/activation/config/KafkaOffsetsMetric.java]: @tsardima seems i saw similar MetricSet in another project.
Should we move metrics to commons project? Or we need to try them separately first? 
@ishkurko @sergeysenja1992 what do you think?

Comment (sergeysenja1992) [reply src/main/java/com/icthh/xm/tmf/ms/activation/config/KafkaOffsetsMetric.java]: @tsardima maybe @amedvedchuk right, and we nedd to made configurable this class in commons

Comment (sergeysenja1992) [reply src/main/java/com/icthh/xm/tmf/ms/activation/config/KafkaOffsetsMetric.java]: like metrics class receive topic and group name

Comment (amedvedchuk) [reply src/main/java/com/icthh/xm/tmf/ms/activation/config/KafkaOffsetsMetric.java]: may be not only kafka related, but also other metrics whose can be common for different microservices.

Comment (sergeysenja1992) [reply src/main/java/com/icthh/xm/tmf/ms/activation/config/KafkaOffsetsMetric.java]: too many dependecy of kafka in gradle, now better to don't move this class to commons
 in future if this mertics will be used in more places, then we will create common project for this metric

**Pull request merged**